manifests/i/IridiumIO/CompactGUI/3.2.1/IridiumIO.CompactGUI.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: IridiumIO.CompactGUI
+PackageVersion: 3.2.1
+InstallerType: exe

```

```suggestion
InstallerType: portable
```

Hi @GhoulBoii,

Sorry for so many separate updates. This appears to be a `portable` type package.